.buildkite/scripts/steps/test/ftr_configs.sh

```diff
@@ -11,6 +11,11 @@ if [ "$FTR_CONFIG_GROUP_KEY" == "" ] && [ "$BUILDKITE_PARALLEL_JOB" == "" ]; the
   exit 1
 fi
 
+if [[ "${FTR_GEN_AI:-}" =~ ^(1|true)$ ]]; then
+  echo "FTR_GEN_AI was set - exposing LLM connectors"
+  export KIBANA_TESTING_AI_CONNECTORS="$(vault_get ai-infra-ci-connectors connectors-config)"
+fi
```

Comment:
It's also an option to load these variables conditionally in the `setup_job_env.sh`, where the rest of the env variables are loaded. It might be better, so all env var loading is co-located.

Reply:
Oh I was looking for that file. Yeah, it probably makes more sense to move that here, I will move that block there.

One question though: I made the `KIBANA_TESTING_AI_CONNECTORS` variable allocation conditional to avoid an additional vault get, as I assumed from our previous discussions we wanted to reduce those. But looking at setup_job_env, I see we already kinda have a ton of calls, so I was wondering: do we really need that additional call I'm adding to be conditional? Because in practice, having this always expose would simplify its consumption.

Reply:
Done in https://github.com/elastic/kibana/pull/198000/commits/4f83898493ed8ae325bbb810d02a9f91d57d09a3 - and kept the variable conditional for now